src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -349,6 +351,10 @@ private Future<Void> validateAndHandleInvoiceStatusTransition(Invoice invoice, I
         validator.validateInvoice(invoice, invoiceFromStorage);
         verifyUserHasManagePermission(invoice.getAcqUnitIds(), invoiceFromStorage.getAcqUnitIds(), okapiHeaders);
         verifyUserHasFiscalYearUpdatePermission(invoice.getFiscalYearId(), invoiceFromStorage.getFiscalYearId(), okapiHeaders);
+        //verify if the user can approve the inovice 
```

Comment:
better to rename method and after it code will be self describing and we will not need comment any more,
you can use this nameL verifyUserHasInvoiceApprovePermission.
The same we should do with other method